spec/lib/api/2.0/workflows-spec.js

```diff
@@ -75,23 +72,70 @@ describe('Http.Api.Workflows.2.0', function () {
 
     describe('workflowsGet', function () {
         it('should return a list of persisted graph objects', function () {
-            var graph = { name: 'foobar' };
+            var graph = 
+                {
+                    "id": "foobar",
+		    "injectableName": "foobar",
```

Comment:
Still there are a lot indent errors.
@srinia6 : There is some tool can beautify the code automatically, this will save you time to check every line, such as http://jsbeautifier.org/.